src/server/lock_manager/config.rs

```diff
@@ -31,6 +31,8 @@ pub struct Config {
     /// assume that the success rate of pessimistic transactions is important
     /// to people who disable the pipelined pessimistic lock feature.
     pub in_memory: bool,
+    pub in_memory_peer_size_limit: ReadableSize,
+    pub in_memory_global_size_limit: ReadableSize,
```

Comment:
Is `in_memory_instance_size_limit` a better name?

Reply:
The related variables are renamed.